### ISSUE TITLE
PSR12.Files.FileHeader.SpacingAfterBlock errors with "use function"

### ISSUE THREAD

**Issue by Zerogiven** [squizlabs/PHP_CodeSniffer#3658]
**Describe the bug**
I got a false positive message cause first of all, there is no file header, second it tells me an error on line 11 where the "use function" happens.

**Code sample**
```php
<?php

declare(strict_types=1);

namespace Idsolutions\SilverStripe\Shop\Article\Control;

use Idsolutions\App\Control\IdsPageController;
use Idsolutions\SilverStripe\Shop\Article\Model\Article;
use SilverStripe\Security\PermissionProvider;
use SilverStripe\View\ArrayData;
use Teapot\StatusCode;
use function assert;

/**
 * Description of Article controller.
 */
class ArticleController extends IdsPageController implements PermissionProvider
{
```

**Custom ruleset**
Nothing for this rule

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```bash
FILE: /home/christopher/Coding/Idsolutions/0_silverstripe_shop/module-shop-article/src/Control/ArticleController.php
--------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------------
 11 | ERROR | [x] Header blocks must be separated by a single blank line
    |       |     (PSR12.Files.FileHeader.SpacingAfterBlock)
```

**Expected behavior**
No error should happen cause it has nothing to do with the file header if i have no spacing between different "use"s.

**Versions (please complete the following information):**
 - OS: x86_64 GNU/Linux
 - PHP: 8.0
 - PHPCS: 3.7.1
 - Standard: PSR12, Slevomat Coding Standard

**Comment by jrfnl** [comment]
PSR12 calls the whole "declare/namespace/use statement" part a file header and demands one blank line between each "block", including one blank line between `use` statements for classes/namespaces and `use` statements for `functions`.

The sniff is working correctly.

See: https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements

**Comment by Zerogiven** [comment]
Big sorry, my mistake, i've read over those few lines inside psr-12 :roll_eyes: 

- One or more class-based use import statements.
- One or more function-based use import statements.
- One or more constant-based use import statements.
